scripts/anvil_tools/file_exists_checker.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
```

Comment:
could you add a "usage" section? that checks if you don't give it the needed arguments, it prints out how to call the script? see the set_requester_pays.sh script for an example